src/components/RadioBorderedBox/__tests__/__snapshots__/index.tsx.snap

```diff
@@ -507,7 +507,7 @@ exports[`RadioBorderedBox renders correctly with label desc and badge 1`] = `
   width: -moz-fit-content;
   width: fit-content;
   background-color: #ef5774;
-  color: #fff;
+  color: #ffffff;
```

Comment:
Why ? 🤔

Reply:
Because:

```js
colorsDeprecated: {
   white: = '#fff'
}

colors: {
  neutral: {
    backgroundWeak: '#ffffff'
  }
}
```

Reply:
![image](https://user-images.githubusercontent.com/62407/144871991-6fca4200-be6c-40e5-897b-8c3592f1d073.png)

Reply:
Yes, why not keep #fff ? :)

Reply:
I wanted to have consistency in terms of how to write colors but actually I can put old way and we will change it later